foreward-text.Rmd

```diff
@@ -0,0 +1,63 @@
+# Foreward {-}
+
+Roger D. Peng
+
+*Johns Hopkins Bloomberg School of Public Health*
+
+January 4, 2022
+
+The field of data science has expanded and grown significantly in recent years, 
+attracting excitement and interest from many different directions. The demand for introductory
+educational materials has grown concurrently with the growth of the field itself, leading to
+a proliferation of textbooks, courses, blog posts, and tutorials. This book is an important
+contribution to this fast-growing literature, but given the wide availability of materials, a
+reader should be inclined to ask, ”What is the unique contribution of this book?” In order
+to answer that question it is useful to step back for a moment and consider the development
+of the field of data science over the past few years.
+
+When thinking about data science, it is important to consider two questions: “What is
+data science?” and “How should one do data science?” The former question is under active
+discussion amongst a broad community of researchers and practitioners and there does
+not appear to be much consensus to date. However, there seems a general understanding
+that data science focuses on the more “active” elements—data wrangling, cleaning, and
+analysis—of answering questions with data. These elements are often highly problem-
```

Comment:
the hyphen here in "problem-specific" has broken over a newline

Please check for this kind of copy-paste error throughout, assuming you just copy-pasted from the pdf